x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/fields/create_field/create_field.tsx

```diff
@@ -253,6 +253,8 @@ export const CreateField = React.memo(function CreateFieldComponent({
 
               <FormDataProvider pathsToWatch={['type', 'subType']}>
                 {({ type, subType }) => {
+                  if (!type) return null;
```

Comment:
I was wondering since we are already touching this file if we could also fix the `<FormDataProvider>` deprecation. I think you can simply change line `87` of this file to `const [{ type, subType }] = useFormData({ form, watch: ['type', 'subType'] });` and remove the `FormDataProvider ` components.

Reply:
fixed, remove the `<FormDataProvider>`  components in this file

Reply:
Awesome, work thanks!